posthog/queries/util.py

```diff
@@ -103,8 +103,13 @@ def convert_to_datetime_aware(date_obj):
     return date_obj
 
 
-def correct_result_for_sampling(value: int, sampling_factor: Optional[float]) -> int:
-    if not sampling_factor:
+def correct_result_for_sampling(value: int, sampling_factor: Optional[float], entity_math: Optional[str] = None) -> int:
+    from posthog.queries.trends.util import ALL_SUPPORTED_MATH_FUNCTIONS
+
+    # We don't adjust results for sampling if:
+    # - There's no sampling_factor specified i.e. the query isn't sampled
+    # - The query performs a math operation because math operations on sampled data yield results in the correct format
```

Comment:
They yield correct results for all statistical functions EXCEPT `sum`, right? That one should still be extrapolated

Reply:
Correct - I wrote this on #12908 and forgot to implement here :D

will push a follow up

Reply:
https://github.com/PostHog/posthog/pull/14473